nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/test/groovy/org/apache/nifi/fingerprint/FingerprintFactoryGroovyIT.groovy

```diff
@@ -90,7 +91,7 @@ class FingerprintFactoryGroovyIT extends GroovyTestCase {
         logger.info("Read initial flow: ${initialFlowXML[0..<100]}...")
 
         // Create the FingerprintFactory with collaborators
-        FingerprintFactory fingerprintFactory = new FingerprintFactory(mockEncryptor, extensionManager)
+        FingerprintFactory fingerprintFactory = new FingerprintFactory(mockEncryptor, extensionManager, SecureHasherFactory.getSecureHasher())
```

Comment:
It looks like this unit test has access to `nifi.properties`, so recommend adjusting to pass the `algorithm` property to `SecureHasherFactory`.